platform-sdk/swirlds-common/src/main/java/com/swirlds/common/threading/framework/internal/AbstractQueueThreadConfiguration.java

```diff
@@ -58,6 +62,16 @@
 
     private QueueThreadMetricsConfiguration metricsConfiguration;
 
+    /**
+     * The callback to run periodically when the queue is idle.
+     */
+    private Runnable idleCallback;
```

Comment:
this should probably be an `InterruptableRunnable`

Reply:
good idea, done